packages/main/src/ToggleButton.ts

```diff
@@ -44,7 +44,14 @@ class ToggleButton extends Button {
 	@property({ type: Boolean })
 	pressed!: boolean;
 
-	_onclick() {
+	_onclick(e: MouseEvent) {
+		if (this.disabled) {
+			e.preventDefault();
+			e.stopPropagation();
+
+			return;
```

Comment:
Thank you for addressing the prev comment. 

Now that I also saw the styles change a bit more carefully

before:
```css
:host([disabled]:active) {
	pointer-events: none;
}
```

after:
```
:host([disabled]) {
	pointer-events: none;
}
```

I wonder if clicking on disabled item would enter the `_onclick `handler at all, would not the `pointer-events:none` prevent that?

Reply:
It does enter the onclick when we click the disabled button, which feels kind of weird, but it does hit breakpoint in the handler and executes the code, which is stopped by a check for being disabled in selectItem.